dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/OracleStatsConnector.java

```diff
@@ -30,19 +31,39 @@
 @ParametersAreNonnullByDefault
 public class OracleStatsConnector extends AbstractOracleConnector {
 
+  static int MAX_DAYS = 10000;
```

Comment:
I would suggest expressing the number of max days as `Duration`.

Reply:
Done, I added the constant `maxDays`.

Reply:
Constants in java follow the UPPER_SNAKE_CASE convention. I would suggest renaming to `MAX_DURATION`. Also this constant can be used directly in the error message. No need to keep the `MAX_DAYS` one.

Reply:
Done